spec/index.bs

```diff
@@ -1224,6 +1236,28 @@ The {{HTMLScriptElement/src}} setter steps are:
     `script`.</ins>
 1.  <ins>Set [=this=]'s [=src=] content attribute to |value|.</ins>
 
+#### Setting slot values from parser #### {#setting-slot-values-from-parser}
+
+This document modifies the HTML parser to set the [=HTMLScriptElement/script text=] value when the script is created.
+
+Modify the [=The text insertion mode=] algorithm as follows:
```

Comment:
Need to work out if this only for HTMLScriptElement's or if it somehow applies to SVG too.